tests/test_api_urls.py

```diff
@@ -42,11 +48,17 @@ def test_api_issues_search(self):
     def test_api_issues_out_of_range(self):
         '''API issue for a non existent number returns JSON 404.'''
         # If we reach 1,000,000 webcompat issues we can celebrate
-        rv = self.app.get('/api/issues/1000000', environ_base=headers)
-        json_body = json.loads(rv.data)
-        self.assertEqual(rv.status_code, 404)
-        self.assertEqual(rv.content_type, 'application/json')
-        self.assertEqual(json_body['status'], 404)
+        with patch('requests.get') as github_data:
+            api_response = MagicMock(spec=Response)
```

Comment:
It might be good to create a helper method here that returns a `MagicMock` object so you don't have to repeat yourself as much in the tests:

``` python
def get_mock(dict):
       blah = MagicMock(spec=Response)
       blah.default_junk = 'whatever is shared'
       for k, v in dict:
           setattr(blah, k, v)
       return blah
```
Then you can do something like `api_response = get_mock({'status_code: 404', .....)`